package.json

```diff
@@ -47,18 +48,15 @@
     "semver": "latest"
   },
   "devDependencies": {
-    "ajv": "^6.5.4",
+    "ajv": "^6.7.0",
```

Comment:
yet again, let's not update unrelated dependencies.

Reply:
I updated only dev dependencies to bring them up to date. Did not deem that dangerous. I'll roll them back.